prover/prover_fri_types/src/lib.rs

```diff
@@ -146,17 +148,21 @@ impl ProverServiceDataKey {
 
     pub fn all_boojum() -> Vec<ProverServiceDataKey> {
         let mut results = vec![];
-        for numeric_circuit in
-            BaseLayerCircuitType::VM as u8..=BaseLayerCircuitType::L1MessagesHasher as u8
+        for numeric_circuit in (BaseLayerCircuitType::VM as u8
+            ..=BaseLayerCircuitType::Secp256r1Verify as u8)
+            .chain(once(BaseLayerCircuitType::EIP4844Repack as u8))
```

Comment:
ouch, do we rely here that all the needed circuits have ids between `VM` and `Secp256r1Verify`? I see it wasn't introduced in the PR, but seems like something to address later on - unless I'm missing something

Reply:
I think this should be exposed from `BaseLayerCircuitType`/`ZkSyncRecursionLayerStorageType` definition.